webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/AnsibleHostInventory.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../node_modules/@patternfly/patternfly/base/patternfly-variables';
+
+.ansible-host-inventory {
+  background-color: $pf-color-white;
```

Comment:
Does the background-color rule have any effects on the tab? I haven't noticed any difference when I removed the rule...